FinderCompass/src/main/java/atomicstryker/findercompass/common/DefaultConfigFilePrinter.java

```diff
@@ -290,8 +292,14 @@ else if (settingList.isEmpty())
 			} else {
 				int[] ids;
 				for(int damage = 0; (ids=OreDictionary.getOreIDs(new ItemStack(pBlock, 1, damage))).length > 0; damage++){

```

Comment:
Sorry, i am currently swamped by pesky real life and not giving this the attention it deserves. Yes, this sounds like a good idea. I don't expect any sizable amount of users to actually use -1, but still, breaking backwards compatibility is bad.